domains/models.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+import uuid
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+AUTHORIZATION_STATUS_CHOICES = (
```

Comment:
This list is taken from the [ACME protocol Authorization Object](https://letsencrypt.github.io/acme-spec/#rfc.section.5.3) status list.